src/main/java/org/opentripplanner/middleware/controllers/api/MonitoredTripController.java

```diff
@@ -75,4 +77,18 @@ private void verifyBelowMaxNumTrips(String userId, Request request) {
             );
         }
     }
+
+    /**
+     * Checks that the given {@link MonitoredTrip} can be monitored
+     * (the underlying {@link org.opentripplanner.middleware.otp.response.Itinerary} has transit and no rentals).
+     */
+    private void checkItineraryCanBeMonitored(MonitoredTrip trip, Request request) {
```

Comment:
Might be better to just accept the Itinerary as an arg. Better name might be `validateItinerary`.

Reply:
... except that there is no real validation or issue reporting performed.

Reply:
Renamed to `checkTripCanBeMonitored` in https://github.com/ibi-group/otp-middleware/pull/91/commits/2c07661450ae3c3531b35660a0892ab10bb6d534...